nilearn/decoding/space_net.py

```diff
@@ -759,6 +759,11 @@ def fit(self, X, y):
         X, y = check_X_y(X, y, ['csr', 'csc', 'coo'], dtype=np.float,
                          multi_output=True, y_numeric=not self.is_classif)
 
+        if not self.is_classif and np.all(np.diff(y) == 0.):
+            raise ValueError("The given input 'y' must have atleast 3 targets"
+                             " to do regression analysis. You provided only"
```

Comment:
PEP8 recommends single quotes for strings, except when enclosing single quotes and in triple quotes of docstrings.

Reply:
In my PEP8 editor it doesn't complain. So, I am fine with I have written.

Reply:
Done

Reply:
I will integrate this one with others if I receive anything.